change/@ni-nimble-tokens-387d48fd-bf9e-4c3d-8d40-e7c58687f8e6.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I'm not familiar with the process for pulling in fonts. Did you basically follow the "steps" in the [fonts README](https://github.com/ni/nimble/blob/c6c858b27deee7eb897a0be87f015e8c0fa860a4/packages/nimble-tokens/dist/fonts/assets/README.md)? If not, could you update that with any additional info about where these files come from?

Reply:
Some more explicit README steps might be helpful as well. Could easily miss that there isa  link to the deployed site ready to use from GitHub.